src/config.rs

```diff
@@ -292,4 +276,11 @@ impl BuilderConfig {
                 .unwrap_or(DEFAULT_CONCURRENCY_LIMIT)
         })
     }
+
+    /// Returns the maximum host gas to use for block building based on the configured max host gas coefficient.
+    pub fn max_host_gas(&self, gas_limit: u64) -> u64 {
+        // Set max host gas to a percentage of the host block gas limit
+        ((gas_limit as u128 * (self.max_host_gas_coefficient.unwrap_or(80) as u128)) / 100u128)
```

Comment:
ditto, this should be a constant so there's only 1 place that needs to change if we decide to change the coefficient

Reply:
yeah i'm happy with that, I think it's safer that way and makes other builders understand that changing this means you're running modified software